engines/cge2/talk.h

```diff
@@ -69,6 +69,7 @@ class Talk : public Sprite {
 public:
 	uint8 *_color;
 
+	const char *_lastText = "";
```

Comment:
This variable can be moved to the `InfoLine` class.
Also it should be initialized in the talk.cpp file and not here. And it can probably be initialized to a null pointer and not an empty string (see the `_oldText` variable in the InfoLine class).